pkg/apis/autoscaling/v1alpha1/pa_types.go

```diff
@@ -92,8 +93,11 @@ type PodAutoscalerSpec struct {
 	// load balances over the pods referenced by the ScaleTargetRef.
 	ServiceName string `json:"serviceName"`
 
-	// The application-layer protocol. Matches `RevisionProtocolType` set on the owning revision.
-	ProtocolType servingv1alpha1.RevisionProtocolType
+	// The application-layer protocol. Matches `ProtocolType` inferred from the revision spec.
+	ProtocolType netv1alpha1.ProtocolType
+
+	// Selector is the pod selector for this revision.
+	Selector map[string]string
```

Comment:
Is this change intended? Wasn't this discussed in the last PR?

Reply:
+1

Reply:
No it's not expected. I'm not sure how it came to be. Probably dirty master